R/tm_g_scatterplot.R

```diff
@@ -432,19 +432,19 @@ ui_g_scatterplot <- function(id, ...) {
             shinyjs::hidden(checkboxInput(ns("show_form"), "Show formula", value = TRUE)),
             shinyjs::hidden(checkboxInput(ns("show_r2"), "Show adj-R Squared", value = TRUE)),
             uiOutput(ns("num_na_removed")),
-            div(
+            tags$div(
               id = ns("label_pos"),
-              div(strong("Stats position")),
-              div(class = "inline-block w-10", helpText("Left")),
-              div(
+              tags$div(tags$strog("Stats position")),
```

Comment:
The typo over here caused the [Exploratory app to fail](https://github.com/insightsengineering/teal.gallery/actions/runs/8304983887/job/22733338936). e2e tests FTW!
The PR #711 will fix this.

Reply:
oops 😱

Reply:
nice